### ISSUE TITLE
Accessibility: Color Contrast Header Links

### ISSUE THREAD

**Issue by danielgroner** [xz/new.css-site#2]
The links in the header fail some contrast ratio accessibility tests, for example Lighthouse Audit in Chrome Dev Tools. Might be a good thing to tweak this.
